syfertext/doc.py

```diff
@@ -136,22 +132,45 @@ def get_encrypted_vector(self, *workers, crypto_provider=None, requires_grad=Tru
 
         for token in self:
 
-            # Get the encypted vector of the token if one exists
+            # Get the encrypted vector of the token if one exists
             if token.has_vector:
 
                 # Increment the vector counter
                 vector_count += 1
 
-                # cumulate token's vector by summing them
+                # Cumulate token's vector by summing them
                 vectors = token.vector if vectors is None else vectors + token.vector
 
-        # if no tokens with vectors were found, just get the default vector (zeros)
+        # If no tokens with vectors were found, just get the default vector(zeros)
         if vector_count == 0:
             doc_vector = self.vocab.vectors.default_vector
         else:
             # Create the final Doc vector
             doc_vector = vectors / vector_count
 
+        return doc_vector
+
+    def get_encrypted_vector(
+        self, *workers, crypto_provider=None, requires_grad=True,
+    ):
+        """Get the mean of the vectors of each Token in this documents.
```

Comment:
Please add type hinting, we use the `typing` library for this:

example

```
crypto_provide: BaseWorker = None
requires_grad: bool = True,
```

Reply:
Should I also add type hinting for *workers as
`*workers : List[BaseWorker]`
??

Reply:
@bicycleman15 it would be enought as `*workers: BaseWorker`